src/taco-cli/cli/utils/vsEmulator.ts

```diff
@@ -0,0 +1,253 @@
+/**
+  *******************************************************
+  *                                                     *
+  *   Copyright (C) Microsoft. All rights reserved.     *
+  *                                                     *
+  *******************************************************
+  */
+
+/// <reference path="../../../typings/node.d.ts" />
+/// <reference path="../../../typings/Q.d.ts" />
+/// <reference path="../../../typings/validator.d.ts" />
+/// <reference path="../../../typings/tacoUtils.d.ts" />
+
+"use strict";
+
+import fs = require("fs");
+import os = require("os");
+import path = require("path");
+import child_process = require("child_process");
+import Q = require("q");
+import validator = require("validator");
+import resources = require("../../resources/resourceManager");
+import utils = require("taco-utils");
+import Logger = utils.Logger;
+
+/**
+ * Handles the Visual Studio Emulator for Android specific operations.
+ */
+export class VSEmulator {
+
+    private static CLI_PATH: string = process.env["ProgramFiles(x86)"] + "\\Microsoft Emulator Manager\\1.0\\emulatorcmd.exe";
```

Comment:
Per our discussion, I will add 32-bit support if we decide to merge this to master.